tools/atip/tools/webdriver.xw_tizen.json

```diff
@@ -11,5 +11,5 @@
         "device": "", 
         "name": "xw_tizen"
     }, 
-    "url-prefix": "file:///opt/home/app/.config/xwalk-service/applications/TEST_APP_ID/"
```

Comment:
@zqzhang  thanks for notify, let's use "TESTER-HOME-DIR" to replace "home/app/...", but not "/home/app/..", otherwise, it will bring risk in /opt/home/app/